homeassistant/config.py

```diff
@@ -361,6 +378,24 @@ def _validate_currency(data: Any) -> Any:
             vol.Optional(CONF_COUNTRY): cv.country,
             vol.Optional(CONF_LANGUAGE): cv.language,
             vol.Optional(CONF_DEBUG): cv.boolean,
+            vol.Optional(CONF_WEBRTC): vol.Schema(
+                {
+                    vol.Required(CONF_ICE_SERVERS): vol.All(
+                        cv.ensure_list,
+                        [
+                            vol.Schema(
+                                {
+                                    vol.Required(CONF_URL): vol.All(
+                                        cv.ensure_list, [stun_or_turn_url]
+                                    ),
+                                    vol.Optional(CONF_USERNAME): cv.string,
+                                    vol.Optional(CONF_CREDENTIAL): cv.string,
+                                }
+                            )
+                        ],
+                    )
+                }
+            ),
```

Comment:
It seems a bit unwanted to add the WebRTC specific validation here and a WebRTC specific object to hass.config.
Maybe a webrtc integration, which just holds the configuration, would be more appropriate?

Reply:
Discussed offline, with the conclusion it's OK to add this to the core config